docs/user_guide/plotting.md

```diff
@@ -1,11 +1,341 @@
+:width: 100%
+:::
+::::
+
+
+:::{dropdown} Where do I find that plot?
+You can add ```--print-output 0``` to every task call, which will print the full filename of all
+outputs of the requested task. Alternatively, you can add ```--fetch-output 0,a``` to directly
+copy all outputs of this task into the directory you are currently in.
+Finally, there is the ```--view-cmd``` parameter you can add to directly display the plot during
+the runtime of the task, e.g. via ```--view-cmd evince-previewer```.
+:::
+
+
+The PlotVariables1D task is located at the bottom of our
+[task graph](https://github.com/columnflow/columnflow/wiki#default-task-graph), which means that
+all tasks leading to PlotVariables1D will be run for all datasets corresponding to the
+```--processes``` we requested using the Calibrators, Selector, and Producers
+(often referred to as CSPs) as requested. In the following examples, we will skip the
+```--calibrators```, ```selector``` and ```producers``` parameters, which means that the defaults
+defined in the config will be used automatically. Examples on how to
+implement your own CSPs can be found in the [calibrators](building_blocks/calibrators),
+[selectors](building_blocks/selectors), and [producers](building_blocks/producers) sections of the
+user guide. The ```--variables``` parameter defines, for which variables we want to create histograms
+and plots. Variables are order objects that need to be defined in the config as shown in the
+[config objects](building_blocks/config_objects) section and the column corresponding to the expression
+statement needs to be stored either after the {py:class}`~columnflow.tasks.reduction.ReduceEvents`
+For each of the category given with the ```--categories``` parameter, one plot will be produced.
+A detailed guide on how to implement categories in Columnflow is given in the
+[categories](building_blocks/categories) section.
+
+To define, which processes and datasets to consider when plotting, you can use the ```--processes```
+and ```--datasets``` parameter. When only processes are given, all datasets corresponding to the
+requested processes will be considered. When only datasets are given, all processes in the config
+will be considered.
+The ```processes``` parameter can be used to change the order of processes in the stack and the legend
+(try for example ```--processes st,tt``` instead) and to further distinguish between sub-processes
+(e.g. via ```--processses, tt_sl,tt_dl,tt_fh```).
+
+:::{dropdown} Note! Do not add the same dataset via multiple processes!
+At the time of writing this documentation, there is still an issue present that histograms corresponding
+to a dataset can accidentally be used multiple times. For example, when adding ```--processes tt,tt_sl```,
+the events corresponding to the dataset ```tt_sl_powheg``` will be displayed twice in the resulting
+plot.
+:::
+
+
+## Customization of plots
```

Comment:
The customiztion of plots is also applicable for 2D and cutflow plots, right? If this is correct, we might want to consider moving this section to the end of the page

Reply:
Yes, the parameters are mostly the same between tasks, but my idea for the guide was to first present all the customization options based on the PlotVariables1D task and only afterwards present the other types of tasks. But I can add one or two sentences to mention that the parameters are mostly the same between tasks.

Reply:
true, added now